config/webpack.dev.js

```diff
@@ -90,7 +96,7 @@ module.exports = {
       // in development "style" loader enables hot editing of CSS.
       {
         test: /\.css$/,
-        loader: 'style!css!postcss'
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader', 'postcss-loader')
```

Comment:
What about not using server side rendering in dev? Or having like a flag? Not having the css being injected when anything changes and have to reload manually it's quite painfully